include/roboteam_ai/world/Field.h

```diff
@@ -185,6 +217,34 @@ class Field {
     // The top most point of the opponents goal (this point is on the right line).
     std::optional<Vector2> theirTopGoalSide;
 
+    // The top position, point with the highest y-coordinate, of the left penalty line (our penalty line).
+    std::optional<Vector2> leftPenaltyLineTop;
+
+    // The bottom position, point with the lowest y-coordinate, of the left penalty line (our penalty line).
+    std::optional<Vector2> leftPenaltyLineBottom;
+
+    // The top position, point with the highest y-coordinate, of the right penalty line (their penalty line).
+    std::optional<Vector2> rightPenaltyLineTop;
+
+    // The bottom position, point with the lowest y-coordinate, of the right penalty line (their penalty line).
+    std::optional<Vector2> rightPenaltyLineBottom;
+
+    /* The bottom left corner of the field, which is the point on the field with the lowest x-coordinate (is a negative value) and lowest y-coordinate (is a negative value) and is
+     * located at our side of the field. */
```

Comment:
obvious comment

Reply:
Obvious comment indeed if you look at the documentation sketch, but less obvious if you do not have this information. In the past I created games where the bottom position had the highest y-coordinate which is also a possible design choice. Since the comment is only 2 lines I guess it hurts more to remove it than to keep it.